tests/debugger/test_debugger_pii.py

```diff
@@ -215,11 +250,21 @@ def setup_pii_redaction_method_full(self):
     )
     @missing_feature(context.library == "nodejs", reason="Not yet implemented", force_skip=True)
     def test_pii_redaction_method_full(self):
-        self._assert(REDACTED_KEYS, REDACTED_TYPES)
+        self._assert(REDACTED_KEYS, REDACTED_TYPES, excluded_identifiers=None)
```

Comment:
@tylfin what I mean is this:

```suggestion
        self._assert(REDACTED_KEYS + EXCLUDED_IDENTIFIERS, REDACTED_TYPES, excluded_identifiers=None)
```